docs/basic-features/image-optimization.md

```diff
@@ -186,6 +186,8 @@ module.exports = {
 }
 ```
 
+If you need to add a `Cache-Control` header for the browser (not recommended), you can configured [`headers`](/docs/api-reference/next.config.js/headers) on the upstream image.
```

Comment:
```suggestion
If you need to add a `Cache-Control` header for the browser (not recommended), you can configure [`headers`](/docs/api-reference/next.config.js/headers) on the upstream image e.g. `/some-asset.jpg` not `/_next/image` itself.
```

Should we stress they shouldn't attempt setting the cache-control header on the `/_next/image` endpoint directly?

Reply:
Sounds good!